Gemfile.lock

```diff
@@ -375,3 +375,6 @@ DEPENDENCIES
   thin
   uglifier (>= 1.0.3)
   unicorn
+
+BUNDLED WITH
```

Comment:
Hopefully this will [only ever result in a warning message](https://meta.discourse.org/t/bundler-always-adds-bundled-with-to-gemfile-lock/29435/5), and looks like something here to stay.  Seem ok to leave?